data/human/outfits.txt

```diff
@@ -298,7 +306,43 @@ outfit "Surveillance Pod"
 	"atmosphere scan" 100
 	description "This is Navy technology, a collection of scanners designed to fit perfectly inside one of their surveillance drones. Since the outputs are fairly standard, you could connect it to your own ship's systems if you want. Installing more than one increases the scan range and speed."
 
+outfit "In-flight Mass Display"
+	category "Heads Up Displays"
+	cost 5000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"mass display" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Some pilots find it useful to know what their ship's current mass is while in-flight, and this little display adds that information to their Heads Up Display."
+
+outfit "Hyperjump Fuel Display"
+	category "Heads Up Displays"
+	cost 11000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"hyperdrive fuel cost display" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump."
 
+outfit "Scram drive Fuel Display"
```

Comment:
One more thing: should Scram drive be capitalized? I think it makes sense for this (and the Jump Drive one) not to be.

Reply:
I'm wondering if this outfit name and the Jump drive one should be like this:
```suggestion
outfit "Scram Drive Fuel Display"
```
You seem to be agreeing with that in your comment?